app/main.html

```diff
@@ -21,6 +21,6 @@
     <div role="app"></div>
 
     <script type="application/javascript" src="node_modules/react-mdl/extra/material.min.js"></script>
-    <script type="application/javascript" src="main.js"></script>
+    <script type="application/javascript" src="app.js"></script>
```

Comment:
Is main.js no longer needed? Why not changing main.js instead of introducing app.js(x)?